pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -2340,4 +2342,20 @@ public void errorIsRaisedIfStaticNodesAreNotWhitelisted() throws IOException {
     assertThat(commandErrorOutput.toString())
         .contains(staticNodeURI.toString(), "not in nodes-whitelist");
   }
+
+  @Test
+  public void pendingTransactionRetentionPeriod() {
+    final int pendingTxRetentionHours = 999;
+
+    parseCommand("--tx-pool-retention-hours", String.valueOf(pendingTxRetentionHours));
+
+    verify(mockControllerBuilder).pendingTransactionRetentionPeriod(intArgumentCaptor.capture());
+
+    assertThat(intArgumentCaptor.getAllValues().get(0)).isEqualTo(pendingTxRetentionHours);
```

Comment:
You don't need to use the int argument captor here - the `verify(mockControllerBuilder).pendingTransactionRetentionPeriod(eq(pendingTxRetentionHours));` line below is enough to assert that the method was called with the argument we expect.  You'd only use an argument captor when you don't know what the argument will be but want to check it's the same value used in some other place.